### PR TITLE
ENH added i/o stuff

### LINKED ISSUE #31: finish the firecrown statistic io
Right now each log-likelihood section returns any data as a set of possibly nest dictionaries containing values which are numpy structured arrays. We should add some temporary code to render this into a set of directories (following the nesting of the dictionaries) with CSV outputs for each record array. 

To be more specific, something like

```python
{'a': {'carr': <structured array>}, 'darr': <structured array>}
```

should end up as 

```bash
output_<analysis_id>/a/carr.csv
output_<analysis_id>/darr.csv
```

This will have to be revisited, but should work for now.

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds i/o.

Closes #31

### CODE REVIEW

File: bin/firecrown [hunk 1]
```diff
@@ -49,6 +50,7 @@ def run(action, config, output):
     # we associate a unique id with each analysis
     # let's write that out with some extra info
     write_metadata(analysis_id, output, config)
+    write_statistics(analysis_id, output, stats)
```
Comment (joezuntz) [inline bin/firecrown]: stats is currently undefined here if we use the run-cosmosis branch

Comment (beckermr) [reply bin/firecrown]: Ack. Good catch and thank you! Fixed!

**Comment by beckermr** [comment]
@joezuntz changed. Anything else?

**Comment by beckermr** [comment]
bump here - I am cool merging without an official approval too 🤷‍♂️

**Comment by beckermr** [comment]
@joezuntz Elisabeth is OK merging here, so I am going to go ahead. Happy to revisit in future PRs if you have further comments!

**Pull request merged**